insights/util/__init__.py

```diff
@@ -93,17 +93,15 @@ def __init__(self, dict_):
         self.__dict__ = dict_
 
 
-def parse_table(content, delim=None, max_splits=-1, strip=True):
+def parse_table(content, delim=None, max_splits=-1, strip=True, header_split=True):
```

Comment:
Based on this information from #534, should the attribute be `header_delim` instead so that you can specify a different header delimiter?  The code at 103 will always split the header, but use either `delim` or ` ` (space).

> parse_table assumes the same delimiter is used for headers as well as the content of the table. This feature makes it so the delimiter can be different between headers and content. It's set to default to the header being the same.
> 
> This is needed for parsing the lssap spec content properly.
```

Reply:
@bfahr Ended up implementing a header_delim and a delim separately. There is actually only one parser using the `delim` argument, so I updated that to use the `header_delim` as well in order to make everything work nicely. All the other parsers that use the `parse_table` function simply use None as the delimiter so everything works.